src/lazy/system_reader.rs

```diff
@@ -77,15 +70,7 @@ const SYMBOLS: RawSymbolTokenRef = RawSymbolTokenRef::SymbolId(7);
 ///# }
 /// ```
 pub struct LazySystemReader<'data, D: LazyDecoder<'data>> {
-    // TODO: Remove this RefCell when the Polonius borrow checker is available.
-    //       See: https://github.com/rust-lang/rust/issues/70255
-    expanding_reader: RefCell<LazyExpandingReader<'data, D>>,
-    // TODO: Make the symbol and macro tables traits on `D` such that they can be configured
-    //       statically. Then 1.0 types can use `Never` for the macro table.
-    symbol_table: SymbolTable,
-    macro_table: MacroTable,
-    allocator: BumpAllocator,
-    pending_lst: PendingLst,
+    pub(crate) expanding_reader: LazyExpandingReader<'data, D>,
```

Comment:
🗺️ The `LazyExpandingReader` can only apply changes to the `EncodingContext` between top-level expressions, which meant it was much easier for it to own all of these resources. Having moved them all over to the `LazyExpandingReader`, it became clear that there wasn't anything left for the `LazySystemReader` to do. I'll merge them in another PR later. Tracking this in #675.